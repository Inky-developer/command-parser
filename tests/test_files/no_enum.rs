use command_parser::parser;

#[parser]
mod my_module {
    #[parse("Foo")]
    struct HelloWorld {}

    #[parse("Bar")]
    struct Bar {}
}

fn main() {}