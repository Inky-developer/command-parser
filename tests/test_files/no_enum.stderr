error: Could not find target enum. The target enum has to be the first enum in the module
 --> $DIR/no_enum.rs:4:1
  |
4 | mod my_module {
  | ^^^
